docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md

```diff
@@ -6,6 +6,8 @@ We have tested building on various environments, including CentOS 6 to 8, Ubuntu
 
 ## Requirements
 
+The following are the configuration requirements for compiling **Nebula Graph**. For the configuration requirements of the operating environment, see [here](../3.configurations/0.system-requirement.md).
```

Comment:
[this](../3.configurations/0.system-requirement.md)) I think it's the suggestion of the topology of the cluster rather than the operating environment requirements